plugins/modules/net_tools/nmcli.py

```diff
@@ -278,7 +278,14 @@
             - The trust level of the connection.
             - When updating this property on a currently activated connection, the change takes effect immediately.
        type: str
-       version_added: 2.0.0
+    wifi_sec:
+       description:
+            - The security configuration of the Wifi connection
```

Comment:
Could you please add some examples for this option? Or maybe add this link for further reference (https://developer.gnome.org/NetworkManager/stable/settings-802-11-wireless-security.html)?
Overally I guess it would be good to improve documentation for this option on how to use.
```suggestion
            - The security configuration of the Wifi connection.
```